statsmodels/tsa/stattools.py

```diff
@@ -1055,6 +1055,115 @@ def arma_order_select_ic(y, max_ar=4, max_ma=2, ic='bic', trend='c',
     return Bunch(**res)
 
 
+def kpss(x, null_hypo="level", lags=None):
```

Comment:
in other cases we just used `trend="c"` with options "c" and "ct", and others if those are available
but adfuller uses `regression : str {'c','ct','ctt','nc'}`

We should move to some consistency in the keywords, arguments